viral/correct_2p_sessions.py

```
@@ -302,18 +341,198 @@ def jb032_2025_04_01(c: SessionCorrection) -> SessionCorrection:
+    return SessionCorrection(
+        epochs=c.epochs,
+        all_tiff_timestamps=c.all_tiff_timestamps,
+        stack_lengths_tiffs=c.stack_lengths_tiffs,
+        chunk_lengths_daq=c.chunk_lengths_daq,
+        frame_times_daq=c.frame_times_daq,
+        offset_after_pre_epoch=0,
+    )
+
+
+@register_correction("JB032", "2025-04-07")
+def jb032_2025_04_07(c: SessionCorrection) -> SessionCorrection:
```

Ruff (0.12.2)

Redefinition of unused `jb032_2025_04_07` from line 240

(F811)